e2e/update-stripe-card-default.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+let page: Page;
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto('/update-stripe-card');
+});
+
```

Comment:
```suggestion
```
Sharing pages between tests is a bit of an anti-pattern. Instead it's fine to simply write a larger test.